sdk/core/azure-core-experimental/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
+## 1.0.0-beta.7 (2020-10-08)
 
+- Updated azure-core released dependency 
```

Comment:
I've made a small change to `JsonPatchDocument` as well, if you don't mind adding a release note for it.

```
- Added APIs to `JsonPatchDocument` which accept pre-serialized JSON.
```